system/yaml/CMakeLists.txt

```diff
@@ -16,11 +16,29 @@
 
 project( yaml_include)
 
-remove_definitions(-fvisibility=hidden)
-
 if(WIN32)
-    set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+    set(LIBYAML_OUTPUT_DIR ${CMAKE_BINARY_DIR}/bin)
+    add_subdirectory(${HPCC_SOURCE_DIR}/system/yaml/libyaml ${CMAKE_BINARY_DIR}/system/libyaml)
```

Comment:
@xwang2713 we'll keep it the old way for windows clienttools builds.  Only need the new logic for linux/osx.

Reply:
OK